packages/aws-cdk-lib/aws-rds/lib/cluster.ts

```diff
@@ -498,6 +527,7 @@ abstract class DatabaseClusterNew extends DatabaseClusterBase {
       deletionProtection: defaultDeletionProtection(props.deletionProtection, props.removalPolicy),
       enableIamDatabaseAuthentication: props.iamAuthentication,
       networkType: props.networkType,
+      storageType: props.storageType?.toString(),
```

Comment:
What happens when you just leave this as `storageType: props.storageType`? You shouldn't need to do `toString()`.

Reply:
It will have a warning for "identifier expected". Here is another [code](https://github.com/aws/aws-cdk/blob/main/packages/%40aws-cdk/aws-apigatewayv2-alpha/lib/common/domain-name.ts#L217) with similar implementation.

Reply:
@corymhall I have updated the code. Please help review.